packages/flutter/lib/src/material/material.dart

```diff
@@ -622,6 +622,7 @@ class _InkHighlight extends InkFeature implements InkHighlight {
   void deactivate() {
     _active = false;
     _alphaController.reverse();
+    super.deactivate();
```

Comment:
(InkFeature is just its own thing, not a widget or render object. It's basically a delegate.)